ProcessLib/HydroMechanics/CreateHydroMechanicsProcess.cpp

```diff
@@ -169,7 +173,7 @@ std::unique_ptr<Process> createHydroMechanicsProcess(
         materialIDs(mesh),     std::move(media_map),
         std::move(solid_constitutive_relations),
         initial_stress,        specific_body_force,
-        mass_lumping};
+        mass_lumping,          output_permeability};
```

Comment:
Other suggestion: instead of  adding output_permeability as a member of `HydroMechanicsProcessData`, and point  `permeability` can be directly used for if-condition. To do this, permeability pointer has to be initialized before creating the instance of  HydroMechanicsProcessData and passed to HydroMechanicsProcessData as

```
      MeshLib::PropertyVector<double>* permeability = output_permeability ? MeshLib::getOrCreateMeshProperty<double>(
            const_cast<MeshLib::Mesh&>(mesh), "permeability",
            MeshLib::MeshItemType::Cell,
            MathLib::KelvinVector::KelvinVectorDimensions<
                DisplacementDim>::value) : nullptr;
```
```
    HydroMechanicsProcessData<DisplacementDim> process_data{
        materialIDs(mesh),     std::move(media_map),
        std::move(solid_constitutive_relations),
        initial_stress,        specific_body_force,
        mass_lumping, permeability};

```